app/views/shared/_notice.html.erb

```diff
@@ -0,0 +1,4 @@
+<div class="alert alert-success alert-dismissible fade show" role="alert">
+    <%= notice %>
```

Comment:
are you sure these indents are ok?

Reply:
Can you be a little clearer on that question?